wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement for the _current version_ of the software available on this website. If there is a conflict between the English language version and any translation, the English version shall apply. Earlier versions may have had different privacy terms that are no longer applicable or supported. Users are encouraged to use the most current version of the Jamulus software.**
+ 
 
 ## Definition of Terms
 
-- "**Server**" The Jamulus Server software, as opposed to the host machine/OS it is running on.
+- "**Server**" The Jamulus software running as a Server and accepting connections from Clients
 - "**Client**" The Jamulus software used to connect to a Server
 - "**Directory**" A Jamulus Server configured to supply a list of Servers to Clients
 
-## Web site
+## Jamulus.io Website
 
-The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. We do not collect your personal data or set tracking cookies. When you connect to the jamulus.io site, your IP is sent to [GitHub Pages](https://pages.github.com/).
+The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. See Github's [privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) for information relating to data collection and privacy. The website at [jamulus.io](https://jamulus.io) does not set tracking cookies.
```

Comment:
Would this be a compromise wording? 

```suggestion
The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. The Jamulus team have not constructed the website to set tracking cookies, but see Github's [privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) for information relating to data collection and privacy overall. 
```

Reply:
I don't know what the issues are. It sounds like there's some way in which we don't fully control the website at jamulus.io.

Reply:
While there is nothing that we **ourselves** put on the website that tracks users, the website is hosted by GitHub. They could easily add tracking code without us being able to stop them.

Reply:
I think that @gilgongo 's wording is good.